modules/cats/src/smithy4s/interopcats/SchemaVisitorHash.scala

```diff
@@ -127,7 +127,8 @@ final class SchemaVisitorHash(
     }
 
     // The encoded form that Hash works against for the eqV method, is a partially-applied curried function.
-    implicit val encoderKInstance = new EncoderK[AltHash, (U => Boolean, Int)] {
+    implicit val encoderKInstance: EncoderK[AltHash] = new EncoderK[AltHash] {
```

Comment:
**thought:** this is a local instance so I didn't annotate it with `Aux`, it'd be useless outside smithy4s